mesonbuild/dependencies/pkgconfig.py

```diff
@@ -514,16 +514,6 @@ def _set_libs(self) -> None:
         raw_libs = self.pkgconfig.libs(self.name, self.static, allow_system=False)
         self.link_args, self.raw_link_args = self._search_libs(libs, raw_libs)
 
-    def get_pkgconfig_variable(self, variable_name: str,
```

Comment:
> It's not obvious to me that we need to drop this function.

It reduces code paths, get_pkgconfig_variable is really just there for historical reasons because we have it in the DSL.

> The behavior is totally different, it's not just a wrapper.

Not totally different, afaik the only difference is it defaults to empty string. Fixed that.